tests/mock.data.js

```diff
@@ -13,10 +13,14 @@ const validVc = require('./validVc.json');
 export const createRequestBody = ({issuer, vc = validVc}) => {
   const {settings: {id, options}} = issuer;
   const credential = klona(vc);
-  credential.issuer = credential?.issuer || id;
-  // convert from millisecond to seconds precision
-  // credential.issuanceDate = createISOTimeStamp();
-  // credential.issuer = id;
+  if (typeof credential.issuer === 'object') {
+    if ("id" in credential.issuer && credential.issuer.id === null) {}
```

Comment:
this could be `if(credential?.issuer?.id)` but the way you wrote it is good and I like it.